### PR TITLE
Fix menu highlighting. Remove redundant menu entry

### PR DESCRIPTION
**Pull request by ErikBrendel**
With this commit, the weird highlighting gets fixed.

It occured, since `users/:id` is a sub-path from `users/:id/dashboard` (for example), and bootstrap thought that we are currently on both pages (which is technically not wrong :D)

I fixed this by **changing the** `users#show` **- URL** from `/users/:id` to `/users/:id/profile`.

-------

In addition to that, the **Menu - Entry "Mein Account" got removed**, since it does exactly the same as "Profil bearbeiten"

### CODE REVIEW

**Comment by GittiHab** [comment]
Why am I added to review this? I am not a developer.

**Comment by ErikBrendel** [comment]
@GittiHab I just wanted to let you know that this issue got addressed. Of course you do not have to review code :D

File: config/routes.rb [hunk 1]
```diff
@@ -36,6 +36,10 @@
   # Sets the devise scope to be used in the controller.
   # http://www.rubydoc.info/github/plataformatec/devise/ActionDispatch%2FRouting%2FMapper%3Adevise_scope
   devise_scope :user do
+    get '/users/:id/profile', to: 'users#show', as: 'user'
+    get '/users/:id', to: redirect('/users/%{id}/profile')
```
Comment (fpottbaecker) [inline config/routes.rb:50]: all of these routes should be member routes, to avoid the necessity of `id`s here

**Review by fpottbaecker — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 1 above)
Comment (ErikBrendel) [reply config/routes.rb:50]: What does this mean? Would you be so kind to adjust it directly on this branch?

**Review by Markus-Brand — APPROVED** [review]

**Review by fpottbaecker — APPROVED** [review]

File: config/routes.rb [hunk 2]
```diff
@@ -41,14 +41,18 @@
   # Sets the devise scope to be used in the controller.
   # http://www.rubydoc.info/github/plataformatec/devise/ActionDispatch%2FRouting%2FMapper%3Adevise_scope
   devise_scope :user do
+    get '/users/:id', to: redirect('/users/%{id}/profile')
```
Comment (fpottbaecker) [inline config/routes.rb]: One note though: Is this redirect necessary, if all links already link to `/users/:id/profile`?

Comment (ErikBrendel) [reply config/routes.rb]: It should not be needed directly. It is convenient, tough (for devs that got used to the url and type it in manually and to keep possible references from the outside intact)

**Pull request merged**